tdrs-backend/tdpservice/users/admin.py

```diff
@@ -16,6 +22,34 @@ class Meta:
         exclude = ['password', 'user_permissions']
         readonly_fields = ['last_login', 'date_joined', 'login_gov_uuid', 'hhs_id']
 
+    def clean(self):
+        """Add extra validation for locations based on roles."""
+        cleaned_data = super().clean()
+        groups = cleaned_data['groups']
+        logger.info(cleaned_data)
+        logger.info(self.instance.groups)
+        if len(groups) > 1:
```

Comment:
Should we change this conditional to `!= 1` to also account for a user without a group? I'm not clear on when or if `clean()` gets called during account creation.

Reply:
I think we should talk about this when we talk about users switching roles. It might be faster to figure it out if we paired on it and walked through the process together.

I didn't think users were supose to get created through the admin panel unless I am misunderstanding the AC.

Reply:
Discussed in sync 2/2.